.github/workflows/update-from-protocol-contracts-sui.yaml

```
@@ -0,0 +1,49 @@
+          path: protocol-contracts-sui
+
+      - name: Update docs
+        run: |
+          mkdir -p src/pages/developers/protocol/
+          rm -f src/pages/developers/protocol/sui.md
+          cp protocol-contracts-sui/docs/gateway.md src/pages/developers/protocol/sui.md
+          rm -rf protocol-contracts-sui
+
+      - name: Check for changes
+        id: check-changes
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)